docs/en/guides/How-to-release.md

```diff
@@ -103,18 +107,25 @@ This script takes care of the following things:
 1. Exclude all unnecessary files in the target source tar, such as `.git`, `.github`, and `.gitmodules`. See the script for more details.
 1. Execute `gpg` and `shasum 512`. 
 
-
 `apache-skywalking-apm-x.y.z-src.tgz` and files ending with `.asc` and `.sha512` may be found in the `tools/releasing` folder.
 
 ## Start the next iteration
 
 Once the release is deployed to the staging repositories, you can start updating the version to the next number and open a pull request.
 
 ```bash
-git checkout next
-./mvnw versions:set-property -DgenerateBackupPoms=false -Dproperty=revision -DnewVersion=<next-version-number>-SNAPSHOT
+# Update the version to the next snapshot version still in the same branch, such as 10.1.0-release
+./mvnw versions:set-property -DgenerateBackupPoms=false -Dproperty=revision -DnewVersion=${NEXT_RELEASE_VERSION}-SNAPSHOT
 git add pom.xml
-git commit -m 'Start next iteration'
+git commit -m "Start next iteration ${NEXT_RELEASE_VERSION}"
```

Comment:
@kezhenxu94 Is this line `NEXT_RELEASE_VERSION` in the commit log correct? Somehow, I don't see this work, but your's for tag commit log is correct.

Reply:
> @kezhenxu94 Is this line `NEXT_RELEASE_VERSION` in the commit log correct? Somehow, I don't see this work, but your's for tag commit log is correct.

IT should work